src/reports/characterpresence.entity.ts

```diff
@@ -3,7 +3,9 @@ import { World, worldArray } from '../constants/world.consts';
 import { Zone, zoneArray } from '../constants/zone.consts';
 import { Faction, factionArray } from '../constants/faction.consts';
 
-@Entity()
+@Entity({
+  name: 'characterpresence'
```

Comment:
Not entirely sure CharacterPresence uses mongo any more... It's all handled in Redis by the Aggregator so I don't think this is used any more, I may have just forgot to delete it out of the websocket.